### PR TITLE
Approximately 3 seconds faster on network drive. Results are identical.

### PR DESCRIPTION
**Pull request by blounsbury**
Note: The scipy documentation states scoreatpercentile will become obsolete and the recommendation is to use numpy percentile.

### CODE REVIEW

**Comment by blounsbury** [comment]
Just realized I should have pushed these commits to separate branches. Now this pull request includes the switch to numpy.percentile and a working continuous integration .travis.yml and requirements.txt file for issue #5.

**Comment by bartnijssen** [comment]
@blounsbury : We haven't revisited `bmorph` recently so I am only seeing these changes now. I'm going to hold off accepting the change till after I can do a test, but the edits are much appreciated.

File: bmorph/core/bmorph.py [hunk 1]
```diff
@@ -66,10 +66,10 @@ def edcdfm(raw_x, raw_cdf, train_cdf, truth_cdf):
            for x in raw_x]
 
     # Given u_t and train_cdf determine train_x
-    train_x = scipy.stats.scoreatpercentile(train_cdf, u_t)
+    train_x = pd.np.percentile(train_cdf, u_t)
 
     # Given u_t and truth_cdf determine truth_x
-    truth_x = scipy.stats.scoreatpercentile(truth_cdf, u_t)
+    truth_x = pd.np.percentile(truth_cdf, u_t)
```
Comment (jhamman) [inline bmorph/core/bmorph.py]: just a note for future reference, you can drop the `pd` (pandas) from these function calls. You are actually use numpy here.

Comment (bartnijssen) [reply bmorph/core/bmorph.py]: I know but I would need to add `import numpy as np` at the top which is not there currently (and I was lazy)

**Pull request merged**